frontend/modules/viewerGui/viewerGui.sagas.ts

```diff
@@ -51,6 +51,20 @@ import {
 } from './viewerGui.selectors';
 
 function* fetchData({ teamspace, model }) {
+	try {
+		yield put(ModelActions.setPendingState(true));
+		const { data: settings } = yield API.getModelSettings(teamspace, model);
+
+		yield put(ModelActions.fetchSettingsSuccess(settings));
+		yield put(ModelActions.setPendingState(false));
+	} catch (e) {
+			if (e.response?.status === 401) {
```

Comment:
I think if there's been an error fetching model settings, we want to show a dialog to redirect to teamspace regardless.

Can we make it so that it's not a `showUnauthorizedModelAccessDialog`, but a `redirectToTeamspaceDialog`

and regardless of what type of error it is here, we show that dialog.

We can pretty much keep the format of endpoint error dialog but with a small change, something like this:

![image](https://user-images.githubusercontent.com/11945337/117162934-1a7fba00-adbb-11eb-98ed-519caacfb871.png)

Reply:
Done